src-built-in/components/toolbar/components/AutoArrange.jsx

```diff
@@ -9,11 +9,16 @@ export default class AutoArrange extends React.Component {
 		};
 		this.bindCorrectContext();
 		let self = this;
-		FSBL.Clients.RouterClient.subscribe('DockingService.AutoarrangeStatus', function (err, response) {
-			self.setState({
-				isAutoArranged: response.data.isAutoArranged
+		FSBL.Clients.LauncherClient.getMonitorInfo({
```

Comment:
@carterbs getMonitorInfo() defaults to the current window's windowIdentifier. I don't think you need to pass it in here.